packages/teams-js/src/private/chat.ts

```diff
@@ -18,64 +18,47 @@ import { ChatMembersInformation } from './interfaces';
  */
 export namespace chat {
   /**
-   * @hidden
-   * Hide from docs
-   * --------------
-   * Allows the user to start or continue a conversation with each subentity inside the tab
    *
+   * Allows the user to start a direct chat with a user
+   * @members = length 1
    * @returns Promise resolved upon completion
    */
-  export function openConversation(openConversationRequest: OpenConversationRequest): Promise<void> {
+
+  export function openChat(openChatRequest: OpenChatRequest): Promise<void> {
     return new Promise<void>(resolve => {
+      if (openChatRequest.members.length !== 1) {
```

Comment:
It would be nice if we could make this "impossible" for app developers to call incorrectly. Maybe if we introduce a second interface that inherits from OpenChatRequest, something like "OpenSingleChatRequest". Then have the new one take only a single member while OpenChatGroupRequest takes an array. If we did that, we probably wouldn't export OpenChatRequest since no app developers should be passing it in (they should be using one of the two interfaces that extend it)